components/header.js

```diff
@@ -0,0 +1,23 @@
+import styled from 'styled-components';
+
+export default function Header() {
+  return (
+    <>
+      <Headerbody>
+        <AppHeadline>MediHelp</AppHeadline>
+        <AbcdeLink>ABCDE</AbcdeLink>
+      </Headerbody>
+    </>
+  );
+}
+
+const Headerbody = styled.header`
+  background-color: hotpink;
+  display: flex;
+  flex-direction: row;
```

Comment:
when assigning the property `display: flex;` the default behaviour is `flex-direction: row;` -> you can remove this line of code :)

Reply:
*solved